packages/edit-site/src/components/sidebar-edit-mode/settings-header/index.js

```diff
@@ -1,67 +1,90 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import { useDispatch } from '@wordpress/data';
+import { useSelect, useDispatch } from '@wordpress/data';
 import { store as interfaceStore } from '@wordpress/interface';
 
 /**
  * Internal dependencies
  */
 import { STORE_NAME } from '../../../store/constants';
 import { SIDEBAR_BLOCK, SIDEBAR_TEMPLATE } from '../constants';
+import { store as editSiteStore } from '../../../store';
 
 const SettingsHeader = ( { sidebarName } ) => {
+	const hasPageContentLock = useSelect( ( select ) =>
+		select( editSiteStore ).hasPageContentLock()
+	);
+
 	const { enableComplementaryArea } = useDispatch( interfaceStore );
 	const openTemplateSettings = () =>
 		enableComplementaryArea( STORE_NAME, SIDEBAR_TEMPLATE );
 	const openBlockSettings = () =>
 		enableComplementaryArea( STORE_NAME, SIDEBAR_BLOCK );
 
-	const [ templateAriaLabel, templateActiveClass ] =
-		sidebarName === SIDEBAR_TEMPLATE
-			? // translators: ARIA label for the Template sidebar tab, selected.
-			  [ __( 'Template (selected)' ), 'is-active' ]
-			: // translators: ARIA label for the Template Settings Sidebar tab, not selected.
-			  [ __( 'Template' ), '' ];
-
-	const [ blockAriaLabel, blockActiveClass ] =
-		sidebarName === SIDEBAR_BLOCK
-			? // translators: ARIA label for the Block Settings Sidebar tab, selected.
-			  [ __( 'Block (selected)' ), 'is-active' ]
-			: // translators: ARIA label for the Block Settings Sidebar tab, not selected.
-			  [ __( 'Block' ), '' ];
+	let templateAriaLabel;
+	if ( hasPageContentLock ) {
+		templateAriaLabel =
+			sidebarName === SIDEBAR_TEMPLATE
+				? // translators: ARIA label for the Template sidebar tab, selected.
+				  __( 'Page (selected)' )
+				: // translators: ARIA label for the Template Settings Sidebar tab, not selected.
+				  __( 'Page' );
```

Comment:
It might be nice to replicate this label in the block breadcrumbs at the bottom, as they still say 'Template' when the sidebar says 'Page'.

Reply:
Good idea!